packages/notifi-frontend-client/lib/models/FusionEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { Types } from '@notifi-network/notifi-graphql';
+
+export type FusionEventMetadata = {
+  uiConfigOverride?: {
+    topicDisplayName?: string;
+    historyDisplayName?: string;
+    icon?: Types.GenericEventIconHint;
+    customIconUrl?: string;
+  };
+  filters: Array<AlertFilter>;
+};
+
+/**
+ * @param name - `string` unique name
+ */
+export type AlertFilter = AlertFilterBase | AlertFrequencyFilter;
+export type AlertFilterBase = {
+  name: string;
+  type: AlertFilterType;
+  executionPriority: number;
+  userInputParams: UserInputParam<UiType>[];
+  staticFilterParams?: Record<string, object | string | number>;
+  requiredParserVariables: Array<RequiredParserVariable>;
+};
+
+export type AlertFrequencyFilter = AlertFilterBase & {
+  minimumDurationBetweenTriggersInMinutes: number;
+};
+
+export type RequiredParserVariable = {
+  variableName: string;
+  variableType: ValueType;
+  variableDescription: string;
+};
+
+export type ValueType = 'integer' | 'price' | 'percentage';
+
+/**
+ * @param UiType - `radio` or `button` (scalable). Define what component should be rendered in Card topic subscription view.
+ * @param defaultValue - The value for default alert subscription
+ */
+export type UserInputParam<T extends UiType> = {
+  name: string;
+  kind: T;
+  valueTypes?: ValueType;
+  options: (string | number)[];
+  defaultValue: string | number;
+  allowCustomInput?: boolean;
+};
+
+export type UiType = 'radio' | 'button';
+type AlertFilterType = 'alertFilter';
```

Comment:
We are currently use 'AlertFilter' | 'SourceFilter' as type of AlertFilterType

Reply:
As discussed, we will keep it as it is now since the 'SourceFilter` is not used anywhere in FE.